packages/bitcore-wallet-service/src/lib/model/txproposal.ts

```diff
@@ -410,18 +414,23 @@ export class TxProposal {
     this._updateStatus();
   }
 
-  sign(copayerId, signatures, xpub) {
+  sign(copayerId, signatures, xpub, bchSchnorr) {
     try {
       // Tests signatures are OK
       const tx = this.getBitcoreTx();
-      ChainService.addSignaturesToBitcoreTx(this.coin, tx, this.inputs, this.inputPaths, signatures, xpub);
+      if(this.coin === 'bch' && this.signingMethod === 'schnorr' && !bchSchnorr) {
+        throw new Error('Need to upgrade - signingMethod not supported');
```

Comment:
I think you need to throw:
      const Errors = require('./errors/errordefinitions');
      const err = Errors.UPGRADE_NEEDED;